### DISCUSSION TITLE
Interface scaling

### DISCUSSION THREAD

**Discussion by Mroczarz** [valnoxy/t480-oc#83] [Q&A]
I need help changing the scale of text. Interface just looks too small, but the setting of scale in Display settings isn't available ( I have 1080p screen). I can just switch the resolution to 1600x900 etc, but I don't want to lose quality, just bigger interface scale.
Is there a possibility to do that?

**Comment by r1ncew** [comment]
Unfortunately Apple only supports native resosution and HiDPI where everything is just doubled. 
1080p is on older iMacs only so you have to live with the original size.
From 1440p you can choose the 1280x720 HiDPI resolution from the scaled option which is displayed in 1440p.
There is no way to change the text size in the GUI itself.

**Reply by ellnic** [reply]
Officially, no. But you can use One Key HiDPI. Despite the project being inactive for some time, I have used it on Ventura and it works well: https://github.com/xzhih/one-key-hidpi/
